minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java

```diff
@@ -222,6 +223,7 @@ private void updateWithParameters(ClusterConfig clusterConfig) {
         for (int i = 0; i < getNumAgents(); i++) {
             MesosAgentConfig agentConfig = (agentConfigs.size() > i) ? agentConfigs.get(i) : new MesosAgentConfig();
             agentConfig.setImageTag(getMesosImageTag());
+            agentConfig.setLoggingLevel(clusterConfig.getLoggingLevel());
```

Comment:
I believe this is wrong: with changes in MesosContainerConfig users can specify logging level on container level. However this code will overwrite it with default or specified level from cluster. This one should work like "number of agents" - if was not specified on container level then take from cluster.